src/components/common/CapabilitiesLeft.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Box, Heading, Paragraph, Image } from 'grommet';
+
+import ScreenshotCard from '../../components/ScreenshotCard.js';
+
+const CapabilitiesLeft = ({ content1, content2, images, labels }) => (
```

Comment:
Let's avoid using '1' and '2' on the named props and instead give them a clearer meaning such as `mainContent` and `subContent`.